packages/hardhat-core/src/internal/core/jsonrpc/types/input/blockTag.ts

```diff
@@ -1,9 +1,5 @@
 import * as t from "io-ts";
 
-// TS2742 workaround
```

Comment:
YESSSSSS

Reply:
This is actually on my to-do list. I don't remember why the workaround was necessary, but the code compiles and works without it, so... is that enough? Can we be confident that this is no longer necessary?

Reply:
Yes, we can, as we are not using it anymore. This happens when there are multiple versions of ethereumjs-util, so that typescript sees multiple definitions of BN. Why in these files? No idea.